xdp-dump/Makefile

```diff
@@ -1,4 +1,5 @@
 # SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
+XDPDUMP_VERSION := 0.1.0
```

Comment:
I was kinda thinking we could just reuse LIBXDP_VERSION for all the tools as well? Seems a bit of hassle to have to maintain several different version series in the same repository?

Reply:
Are we ok with updating the libxdp version if nothing changes in the libraty itself? If so I can use that version (I had that initially).

Reply:
Hmm, that's a good point. I guess we may need to split them in the future. But for now at least, we need to keep them together, if nothing else because of the packaging. But how about we define TOOLS_VERSION := $(LIBXDP_VERSION) and use that as the output in --version, man page etc? Then we can always split the two versions down the line; I think it'll be fine to bump all the tool versions even when only updating one of them...

Reply:
Will add this in the next commit